pkg/airgap/file_scp.go

```diff
@@ -283,3 +302,33 @@ func getScpFileMap(arch string, pkg *common.Package) (map[string]fileMap, error)
 func getRemoteTmpDir(clustername string) string {
 	return filepath.Join(remoteTmpDir, clustername)
 }
+
+func getDataPath(extraArgs string) string {
+	dataPath := defaultDataDirPath
+	args := strings.Split(extraArgs, " ")
+	for i, arg := range args {
+		var prefix string
+		if strings.HasPrefix(arg, dataDirParamPrefix) {
+			prefix = dataDirParamPrefix
+		}
+		if strings.HasPrefix(arg, dataDirParamPrefixShot) {
+			prefix = dataDirParamPrefixShot
+		}
+		if prefix == "" {
+			continue
+		}
+		// if the arg == dataPath prefix, return the next arg
+		if len(arg) == len(prefix) && i < len(args)-1 {
```

Comment:
Is `--data-dir=/data` or `-d /data` also valid for this situation?

Reply:
It should be valid

Reply:
Added test cases